packages/react-server-cli/src/parseCliArgs.js

```diff
@@ -0,0 +1,56 @@
+import yargs from "yargs"
+
+export default (isProduction, args = process.argv) => {
+	return yargs(args)
+		.usage('Usage: $0 [options]')
+		.option("routes", {
+			default: "./routes.js",
+			describe: "The routes file to load.",
+		})
+		.option("p", {
+			alias: "port",
+			default: 3000,
+			describe: "Port to start listening for react-server",
+			type: "number",
+		})
+		.option("jsPort", {
+			default: 3001,
+			describe: "Port to start listening for react-server's JavaScript",
+			type: "number",
+		})
+		.option("h", {
+			alias: "hot",
+			default: !isProduction,
+			describe: "Load the app so that it can be hot reloaded in the browser. Default is false in production mode, true otherwise.",
+			type: "boolean",
+		})
+		.option("m", {
+			alias: "minify",
+			default: isProduction,
+			describe: "Optimize client JS when option is present. Takes a bit longer to compile. Default is true in production mode, false otherwise.",
+			type: "boolean",
+		})
+		.option("loglevel", {
+			default: isProduction ? "notice" : "debug",
+			describe: "Set the severity level for the logs being reported. Values are, in ascending order of severity: 'debug', 'info', 'notice', 'warning', 'error', 'critical', 'alert', 'emergency'. Default is 'notice' in production mode, 'debug' otherwise.",
+			type: "string",
+		})
+		.option("compileonly", {
+			default: false,
+			describe: "Compile the client JavaScript only, and don't start any servers. This is what you want to do if you are building the client JavaScript to be hosted on a CDN. Unless you have a very specific reason, it's almost alway a good idea to only do this in production mode. Defaults to false.",
+			type: "boolean",
+		})
```

Comment:
Good point. Standardized the args in aa5411c so that the command line args are all hyphenated and the JavaScript representations are all camelCased. The changed args are `log-level`, `js-url`, `js-port`, and `compile-only`. Thanks!